--- a/etc/init.d/dlnainit	2019-12-25 04:18:35.000000000 +0100
+++ b/etc/init.d/dlnainit	2021-01-03 12:47:21.885260411 +0100
@@ -37,7 +37,8 @@
     if [ ! -f "/data/dlna/device.xml" ]; then
 	config_device_file
     fi
-    ota_updata_deicie
+    #ota_updata_deicie
+    [ -x /data/enable_dlna ] || return 0
     procd_open_instance
     procd_set_param command /usr/bin/dlna
     procd_set_param respawn
--- a/etc/init.d/mitv-disc	2019-12-25 04:18:35.000000000 +0100
+++ b/etc/init.d/mitv-disc	2021-01-03 12:47:21.965260847 +0100
@@ -7,6 +7,7 @@
 
 start_service() {
   channel=`/sbin/uci -q get /usr/share/mico/version.version.CHANNEL`
+  [ -x /data/enable_dlna ] || return 0
   if [ "$channel" = "release" ]; then
      /etc/init.d/pns mitv_off 1>/dev/null 2>&1
      #return
--- a/etc/init.d/mdplay	2019-12-25 04:18:35.000000000 +0100
+++ b/etc/init.d/mdplay	2021-01-03 12:47:22.005261065 +0100
@@ -5,6 +5,7 @@
 USE_PROCD=1
 
 start_service() {
+    [ -x /data/enable_mdplay ] || return 0
     procd_open_instance
     procd_set_param command /usr/bin/mdplay
     procd_set_param nice
--- a/etc/init.d/pns	2019-12-25 04:18:35.000000000 +0100
+++ b/etc/init.d/pns	2021-01-03 12:47:21.565258666 +0100
@@ -197,6 +197,7 @@
         fi
     fi
     
+    [ -x /data/enable_pns ] || return 0
     local vendor_name=$(/usr/bin/mico_vendor_helper vendor_name_get);
     if [ "nuance" = $vendor_name ]; then
         _start_mipns_nuance
