--- a/etc/asound.conf	2018-06-14 04:17:09.000000000 +0200
+++ b/etc/asound.conf	2021-05-30 00:16:57.858946184 +0200
@@ -1,4 +1,20 @@
 pcm.!default {
+  type asym
+  playback.pcm "general"
+  capture.pcm "mico_record"
+}
+
+pcm.mico_record {
+  type plug
+  slave {
+    pcm "hw:1,0"      
+    channels 4
+    format S16_LE
+    rate 16000
+  }
+}
+
+pcm.general {
     type plug
     slave {
       pcm {
@@ -55,14 +71,7 @@
   }
 }
 
-
 pcm.eq {
-    type mieq
-    channels 2
-    slave.pcm	"dmixer"
-}
-
-pcm.dmixer {
    type dmix
    ipc_key 1024
    slave {
@@ -71,5 +80,3 @@
 	 rate 48000
    }
 }
-
-defaults.pcm.rate_converter "speexrate_medium"
