--- a/networking/udhcp/dhcpc.c	2021-12-26 17:55:58.000000000 +0100
+++ b/networking/udhcp/dhcpc.c	2022-11-28 17:29:02.568323960 +0100
@@ -722,6 +722,7 @@
 static NOINLINE int send_discover(uint32_t requested)
 {
 	struct dhcp_packet packet;
+	static int msgs = 0;
 
 	/* Fill in: op, htype, hlen, cookie, chaddr fields,
 	 * xid field, message type option:
@@ -736,6 +737,7 @@
 	 */
 	add_client_options(&packet);
 
+	if (msgs++ < 3)
 	bb_simple_info_msg("broadcasting discover");
 	return raw_bcast_from_client_data_ifindex(&packet, INADDR_ANY);
 }
