--- a/CMakeLists.txt	2015-02-22 21:53:59.000000000 +0100
+++ b/CMakeLists.txt	2024-01-30 21:18:43.038501694 +0100
@@ -68,7 +68,7 @@
 project(sox C)
 
 if(CMAKE_COMPILER_IS_GNUCC)
-	add_definitions(-fstack-protector -Wall -W -Wmissing-prototypes -Wstrict-prototypes -pedantic -Wno-format -Wno-long-long)
+	add_definitions(-fstack-protector -Wno-format -Wno-long-long)
 endif(CMAKE_COMPILER_IS_GNUCC)
 
 include(TestBigEndian)
