--- a/usr/sbin/boot_function.sh	2023-06-24 19:27:31.852322961 +0200
+++ b/usr/sbin/boot_function.sh	2023-06-24 20:15:40.478070686 +0200
@@ -236,13 +236,14 @@
 
 hostname_set()
 {
-    local myname=$(micocfg_model);
-    echo "${myname:-mico}" > /proc/sys/kernel/hostname
+    SN=$(echo -n `uci -c /data/etc get binfo.binfo.sn` | tail -c -4)
+    MODEL=$(uci -c /usr/share/mico get version.version.HARDWARE)
+    echo "${MODEL}-${SN}" > /proc/sys/kernel/hostname
 }
 
 hostname_set_sn()
 {
-    local myname=$(micocfg_hostname);
+    local myname=$(hostname);
     echo "${myname:-mico}" > /proc/sys/kernel/hostname
 }
 
